var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -38,7 +39,10 @@ class PyMdanalysis(PythonPackage):
     depends_on('py-six@1.4.0:', type=('build', 'run'))
```

Comment:
For `python`, can you add type=build/run?

For `numpy`, it looks like 1.13.3 is required now.